duckduckgo/BrowserServicesKit
# Pass config to Autofill script

Task/Issue URL: https://app.asana.com/0/72649045549333/1201640241287329/f
Tech Design URL: https://app.asana.com/0/481882893211075/1201716942816252/f
CC:

**Description**:


**Steps to test this PR**:
1. Use with this pr: https://app.asana.com/0/0/1201760729306248/f
1. Modify privacy config manager to use embedded config
2. Ensure autofill is enabled in `macos-config.json`
3. Visit https://duckduckgo.com/email/ and ensure you can sign in
4. Visit https://privacy-test-pages.glitch.me/autofill/modal.html and https://privacy-test-pages.glitch.me/autofill/form-submission.html and ensure they behave as expected



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
